guide/payments/coin-selection.md

```diff
@@ -128,30 +128,40 @@ A user chooses to send a payment to one of their contacts. They enter the amount
 
 </div>
 
-A few *popular algorithms* currently implemented by Bitcoin wallets:
+### Examples
 
-- **First In First Out (FIFO) / Last In, First Out (LIFO)**<br/>
+Below are some popular algorithms currently implemented by Bitcoin wallets that optimize for speed and lower fees:
+
+#### First In First Out (FIFO) / Last In, First Out (LIFO)<br/>
```

Comment:
General feedback, the `<br>` makes the hash drop to the next line.

<img width="868" alt="Screenshot 2021-08-06 at 13 11 12" src="https://user-images.githubusercontent.com/183140/128502073-a0b786bd-ce9b-4999-a5ab-e061ae506783.png">

it looks fine without it.

<img width="856" alt="Screenshot 2021-08-06 at 13 11 55" src="https://user-images.githubusercontent.com/183140/128502153-6a27e052-0dff-4e29-b058-13c6ba6b6125.png">

Reply:
Fixed